config/routes.rb

```diff
@@ -5,12 +5,12 @@
 
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show] do
-      resources :comments, only: [:new, :create]
+      resources :comments, only: [:new, :create, :destroy]
       resources :likes, only: [:create]
     end
   end
 
-  resources :posts, only: [:new, :create]
+  resources :posts, only: [:new, :create, :destroy]
```

Comment:
[OPTIONAL] You have duplicated the routes for posts which causes some errors in your web app. It would be better to keep one `resources` for posts so that your code is clean. I recommend deleting line 13 and keeping line 8.